Tests/SentryTests/SentryClientTests.swift

```diff
@@ -214,7 +214,70 @@ class SentryClientTest: XCTestCase {
         
         eventId.assertIsEmpty()
     }
-
+    
+    func test_AttachmentProcessor_CaptureEvent() {
+        let sut = fixture.getSut()
+        let event = Event()
+        var checkEvent = false
```

Comment:
`l`: Wouldn't using a `XCTestExpectation` be more appropriate here? We then could use `XCTAssertEqual(event, e)` inside `TestAttachmentProcessor`, which would give us a better failure message when  `event != e`. Now `XCTAssertTrue(checkEvent)` would fail and you don't immediately know why.